src/mpz.rs

```diff
@@ -256,6 +259,19 @@ impl Mpz {
         }
     }
 
+    pub fn div_rem(&self, other: &Mpz) -> (Mpz, Mpz) {
```

Comment:
I can't work out why you add this? Was there a clippy warning after changing to 2018 that I didn't see?

Reply:
I use it for certain arithmetic operations where I get both results with one call